gobblin-restli/gobblin-flow-config-service/gobblin-flow-config-service-server/src/main/java/org/apache/gobblin/service/FlowExecutionResourceLocalHandler.java

```diff
@@ -215,15 +216,18 @@ private static long getFlowStartTime(org.apache.gobblin.service.monitoring.FlowS
    * @param completionPercentage of the job
    * @return time left in seconds
    */
-  public static long estimateCopyTimeLeft(Long currentTime, Long startTime, int completionPercentage) {
+  public static String estimateCopyTimeLeft(Long currentTime, Long startTime, int completionPercentage) {
     if (completionPercentage == 0) {
-      return 0;
+      return "Not calculated yet";
     }
 
     Instant current = Instant.ofEpochMilli(currentTime);
     Instant start = Instant.ofEpochMilli(startTime);
     Long timeElapsed = Duration.between(start, current).getSeconds();
     Long timeLeft = (long) (timeElapsed * (100.0 / Double.valueOf(completionPercentage) - 1));
-    return timeLeft;
+
+    Long timeLeftInMilli = Instant.ofEpochSecond(timeLeft).toEpochMilli();
```

Comment:
I'm wondering if it makes sense to change the time to String here, or if we want to change it on the API level right before outputting the number to the user. A few minor caveats for doing it here is that strings take more space and that it'll be harder to run queries through the metrics with strings as opposed to a number for timestamps. Thoughts?